src/app/views/home/sections/Landing.js

```diff
@@ -15,7 +15,7 @@ function Landing() {
 					<Fade duration={1000} left>
 						<div className="hero-wrapper">
 							<h1 className="glow">ZotHacks</h1>
-							<span className="h3">November 20-21, 2021</span>
+							<span className="h3">November 13-14, 2022</span>
```

Comment:
Let's avoid putting unrelated changes in the same commit so we can have a cleaner history.

Reply:
Good point. Should we just create separate pull requests for these then?

Reply:
I would say it's okay to have multiple commits in a pull request if they're all thematically related. It's up to your judgement where you want to draw the line. For this change specifically, I would hold off since we are yet to confirm the actual dates of ZotHacks.